epub33/rs/index.html

```diff
@@ -862,21 +862,6 @@ <h4>Metadata</h4>
 							purely advisory. The language information expressed in the resource determines its language
 							for processing and rendering purposes, as defined in the <a
 								href="#confreq-rs-pkg-lang-no-content">internationalization requirements</a>.</p>
-						<p id="sec-linked-records" data-tests="#pkg-linked-records">In the case of a <a
-								data-cite="epub-33#record">linked metadata record</a> [[epub-33]], reading systems MUST
-							NOT skip processing the metadata expressed in the package document (i.e., use only the
-							information expressed in the record). Reading systems MAY compile metadata from multiple
-							linked records.</p>
-						<p id="confreq-rs-pkg-link-order"
-							data-tests="#pkg-linked-records_link-priority,#pkg-linked-records_link-order">When resolving
-							discrepancies and conflicts between metadata expressed in the package document and in linked
-							metadata records, reading systems MUST use the document order of <span data-cite="epub-33"
-								>[^link^]</span> elements [[epub-33]] in the package document to establish precedence
-							(i.e., metadata in the first linked record has the highest precedence and metadata in the
-							package document the lowest, regardless of whether the <code>link</code> elements occur
-							before, within, or after the package metadata elements).</p>
-						<p id="confreq-rs-pkg-link-rendering">Reading systems MUST ignore any instructions contained in
-							linked resources related to the layout and rendering of the EPUB publication.</p>
```

Comment:
This sentence looks a bit different from the previous ones. Was it also discussed that this should go? 

(Although... we do not seem to have any test for it anyway, so we may just well remove it...)

Reply:
It reads like another case of precedence over what might have been expressed in a browser-friendly package document -- don't use the layout behaviours that were defined for a browser rendering.

You can add properties to the package metadata if you control layout and rendering, or create custom attributes, so explicitly banning from linked resources when we no longer expect the bff format seems a bit paranoid. No one's going to through the effort of defining instructions through linked records when the simpler methods exist.